apps/ann-benchmarks/run.py

```diff
@@ -36,15 +36,15 @@
 
 
 parser = argparse.ArgumentParser()
-client = weaviate.connect_to_local()
+client = weaviate.connect_to_local(skip_init_checks=True)
```

Comment:
The commit says temporarily, will this be removed in another patch?

Reply:
Thanks @donomii, I've removed this was used for testing.